### PR TITLE
cart UI is created

### PR DESCRIPTION
**Pull request by AHMED-NASSER-Mohmaed**

### CODE REVIEW

File: client/js/cart2.js
Comment (sarahsalem98) [inline client/js/cart2.js:1]: what is the purpose?

File: client/cart.html
Comment (sarahsalem98) [inline client/cart.html:1]: why there are multiple cart files (please remove any redundant or make it more descriptive)?
or is there availability to make them in one page and use  id   ?

File: client/cart2.html
Comment (sarahsalem98) [inline client/cart2.html:1]: same here in naming ?

File: client/css/cart1.css
Comment (sarahsalem98) [inline client/css/cart1.css:1]: why not put all of them in one file cart.css?

File: client/jquery/jquery.js
Comment (sarahsalem98) [inline client/jquery/jquery.js:1]: move to js folder

File: client/js/cart1.js
Comment (sarahsalem98) [inline client/js/cart1.js:1]: still why not all in one cart.js?

File: client/cart.html
Comment (AHMED-NASSER-Mohmaed) [reply client/cart.html:1]: there's the availability to rename them but not make them in only one file at the period of development

File: client/js/cart1.js
Comment (AHMED-NASSER-Mohmaed) [reply client/js/cart1.js:1]: at production time, not now due to development purpose, i can't work with your approach to make all of them in only one file. we have finished the ui only as a static website , we are going back to add more functionallity to all js files.

**Pull request merged**